caf:  ble_adv: documentation update (#6442)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

  **Workaround:** Use the quick session resume feature only for configurations with the cellular network backend.

Common Application Framework (CAF)
==================================

.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0

> Directed advertising (:kconfig:`CONFIG_CAF_BLE_ADV_DIRECT_ADV`) does not work properly.

Reviewer: You need to add the related JIRA issue for this known issue. The update requires this part as mandatory, as mentioned in the beginning of the known_issues.rst raw file.

  **Workaround:** Use the quick session resume feature only for configurations with the cellular network backend.

Common Application Framework (CAF)
==================================

.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0

> NCSDK-13058: Directed advertising (:kconfig:`CONFIG_CAF_BLE_ADV_DIRECT_ADV`) does not work properly.
>   Device is unable to connect to host, when it uses directed advertising.

Reviewer: ```suggestion
NCSDK-13058: Directed advertising does not work
  The directed advertising feature enabled with the :kconfig:`CONFIG_CAF_BLE_ADV_DIRECT_ADV` option does not work as intended.
  The device is unable to connect to the host when it uses directed advertising.
```

Reviewer 2: Actually this is partially true. The problem affects only peers with RPA that do not support resolution of directed advertisements (we can check this by reading the Central Address Resolution Characteristic). Also the problem is related only to loading settings, so it would not be visible before reboot. To simplify implementation, we considered using direct advertising only for some time and then switch to indirect advertising, See `NCSDK-13128` for details.

Reviewer 2: Maybe we should state that `direct advertising towards privacy enabled peers may not work` to avoid explaining all of the details?

Also, please add `cherry-pick commit_hash` as a workaround. I think that is way more useful information - fix can be easily cherry-picked (then fix needs to go in a separate PR and you can refer to commit hash from main).

Reviewer: Sounds reasonable.

Reviewer: @MarekPieta , what are `privacy-enabled peers`?

Reviewer 2: The peers that enable BLE privacy and use RPA (Resolvable Private Address),

Reviewer: @MarekPieta , @alstrzebonski , then I'd suggest:
```suggestion
NCSDK-13058: Directed advertising does not work
  The directed advertising feature enabled with the :kconfig:`CONFIG_CAF_BLE_ADV_DIRECT_ADV` option does not work as intended.
  Using directed advertising towards peers that enable privacy may result in connection establishing problems
```

>   **Workaround** Do not use directed advertising.

Reviewer: ```suggestion
  **Workaround** Do not use directed advertising in CAF.
```
